joomla/joomla-cms
# [mod_feed] Add date to layout & options

PR for #14300

Adds two new options to the module to optionally display the feed and item dates

### Test instructions
Create a new module to display a feed
To test you can use this feed http://www.feedforall.com/blog-feed.xml

Check to ensure that the TWO date options are set to NO by default and publish the module

Check the module on the front end and you will see a regular field as before this PR

Now enable the Date options and you will see a date for the feed and a date for each item

Also check the admin feed module where the same changes have been made
